presto-resource-group-managers/src/test/java/com/facebook/presto/resourceGroups/TestingResourceGroup.java

```
@@ -153,4 +154,10 @@ public void setJmxExport(boolean export)
     {
         jmxExport = export;
     }
+
+    @Override
+    public ResourceGroupInfo getInfo()
+    {
+        return null;
```

Probably better to throw an `UnsupportedOperationException` here